Makefile

```diff
@@ -58,11 +58,17 @@ FILESYSTEM_PORT ?= 8081
 # Preferred metrics port on your host system
 METRICS_PORT ?= 9118
 
-# Preferred Capella version
-CAPELLA_VERSION ?= 5.2.0
+# Capella version used for builds and tests
+export CAPELLA_VERSIONS ?= 5.0.0 5.2.0 6.0.0
+
+# Capella version used to run containers
+CAPELLA_VERSION ?= 6.0.0
```

Comment:
Is it intended to change the "default" version of capella here from `5.2.0` to `6.0.0`?

Reply:
Yes :)